R/prepare_matches_mapper.R

```diff
@@ -12,10 +12,11 @@ prepare_matches_mapper <- function(mapper, activities) {
   mapper |>
     left_join(activities, by = "activity_uuid_product_uuid") |>
     summarise(across(everything(), ~ paste0(na.omit(unique(.)), collapse = "; ")), .by = c("country", "main_activity", "clustered")) |>
-    ungroup() |>
+    mutate(across(everything(), ~ifelse(. == "", NA_character_, .))) |>
     distinct() |>
+    mutate(multi_match = as.logical(.data$multi_match)) |>
     mutate(multi_match = case_when(
-      multi_match == "" ~ "FALSE",
+      is.na(multi_match) ~ FALSE,
       TRUE ~ multi_match
     ))
```

Comment:
I see only two cases so maybe we can simplify with `ifelse()`?

If so I think the code would be something like this:

```r
  mutate(multi_match = ifelse(is.na(multi_match), FALSE, TRUE)
```

Reply:
Yes make sense! changed